### DISCUSSION TITLE
Seesaw implementation notes

### DISCUSSION THREAD

**Discussion by pelikhan** [microsoft/devicescript#421] [General]
This discussion is about coordinating seesaw support. (It is also an experiment to see if we can get the community to contribute). 

The current implementation is quite "empty" and requires to port various helper function. Chime in if you're interrested to add more info.

* pull request: https://github.com/microsoft/devicescript/pull/420

**Comment by Tanmay53** [comment]
What are the key results which we wish to achieve after seesaw support?

**Reply by pelikhan** [reply]
Adafruit sells a number of board with a seesaw firmware pre-build. Having a seesaw library would allow to use those boards.

https://www.adafruit.com/search?q=seesaw

I would say that this is low priority and mostly useful if someone was in a position where seesaw was needed.
